Onboarding/exploring-tour-reminders.md

```diff
@@ -0,0 +1,85 @@
+<!--
+ dictionary: "config"
+version: "0.1.0"
+columns: [] 
+-->
+
+| Key               | Value                                    |
+| ----------------- |------------------------------------------|
+| contentrepo_token | xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx |
+
+# Reminders
+
+<!-- { section: "fa72e364-2a58-4731-9b27-2179875eaa57", x: 0, y: 0} -->
+
+```stack
+card Reminder do
+  search =
+    get(
+      "https://content-repo-api-qa.prk-k8s.prd-p6t.org/api/v2/pages/",
+      query: [
+        ["slug", "mnch_onboarding_reminder"]
+      ],
+      headers: [["Authorization", "Token @config.items.contentrepo_token"]]
+    )
+
+  page_id = search.body.results[0].id
+
+  page =
+    get(
+      "https://content-repo-api-qa.prk-k8s.prd-p6t.org/api/v2/pages/@page_id/",
+      query: [
+        ["whatsapp", "true"]
+      ],
+      headers: [["Authorization", "Token @config.items.contentrepo_token"]]
+    )
+
+  message = page.body.body.text.value
+  button_labels = map(message.buttons, & &1.value.title)
+
+  buttons(
+    CreateProfiles: "@button_labels[0]",
+    RemindLater: "@button_labels[1]",
+    RemindLater: "@button_labels[2]"
+  ) do
+    text("@message.message")
+  end
+end
+
+card CreateProfiles do
+  run_stack("d5f5cfef-1961-4459-a9fe-205a1cabfdfb")
+end
+
+card RemindLater do
+  search =
+    get(
+      "https://content-repo-api-qa.prk-k8s.prd-p6t.org/api/v2/pages/",
+      query: [
+        ["slug", "mnch_onboarding_reminder_later"]
+      ],
+      headers: [["Authorization", "Token @config.items.contentrepo_token"]]
+    )
+
+  page_id = search.body.results[0].id
+
+  page =
+    get(
+      "https://content-repo-api-qa.prk-k8s.prd-p6t.org/api/v2/pages/@page_id/",
+      query: [
+        ["whatsapp", "true"]
+      ],
+      headers: [["Authorization", "Token @config.items.contentrepo_token"]]
+    )
+
+  message = page.body.body.text.value
+  button_labels = map(message.buttons, & &1.value.title)
+
+  buttons(
+    CreateProfiles: "@button_labels[0]",
+    RemindLater: "@button_labels[1]"
```

Comment:
This will mean that if they click this button, they'll just get this same message again, is that intended? Or is this a TODO that we need to implement the functionality later? If it's the latter, then maybe we can add a comment saying so

Reply:
I will chat with Gerrit i see conflicts between intro welcome and these reminders.